### PR TITLE
Docs: Change "help text" to "helper text" 

### LINKED ISSUE #1402: docs: Consistent terminology. Helper text versus help text
This issue is to address inconsistencies in terminology when referencing the text that appears underneath a field to provide additional context for a user.  It should be referred to as _helper text_, not _help text_. This will make it consistent with the Sketch kit, and happily falls in line with what product designers are calling it. 

Just to clarify, text that sits _inside_ the field is called placeholder text, and that does not change. 

1. Search the site for instances of help text and change to helper text, where needed.
2. Check for images of helper text to ensure that the helper text sits below the field. For example, on this page the image will need to be updated: https://www.carbondesignsystem.com/components/text-input/usage/
We need to check if there are other instances of the old positioning.

### PR DESCRIPTION
**Pull request by jillianhowarth**
Closes #1402 

Changes all instances of "help text" to "helper text" and updates helper text position in images.

### CODE REVIEW

File: src/pages/components/dropdown/usage.mdx [hunk 1]
```diff
@@ -244,16 +244,16 @@ Field containers should vertically align to the grid and with other form compone
...
-- Help text is always available when the dropdown field is focused and appears underneath the label.
+- Helper text is pertinent information that assists the user in choosing the right selection from the dropdown menu.
+- Helper text is always available when the dropdown field is focused and appears underneath the label.
 - Use sentence-style capitalization and write the text as full sentences with punctuation, unless space is limited.
 
 <Row>
 <Column colLg={8}>
 
-![Dropdown assistive labels, help text or none.](images/dropdown-usage-5.png)
+![Dropdown assistive labels, helerp text or none.](images/dropdown-usage-5.png)
```
Comment (aagonzales) [inline src/pages/components/dropdown/usage.mdx:256]: Typo here "helerp"

Comment (janchild) [inline src/pages/components/dropdown/usage.mdx]:
```suggestion
![Dropdown assistive labels, helper text or none.](images/dropdown-usage-5.png)
```

Comment (jillianhowarth) [reply src/pages/components/dropdown/usage.mdx]: Oops good catch. Fixed!

Comment (jillianhowarth) [reply src/pages/components/dropdown/usage.mdx]: I just pushed the fix for that 🙂

**Pull request merged**